e2etest/bgptests/bgp.go

```diff
@@ -433,6 +433,7 @@ var _ = ginkgo.Describe("BGP", func() {
 			Peers: metallb.PeersForContainers(FRRContainers, ipFamily, func(p *metallbv1beta2.BGPPeer) {
 				p.Spec.PasswordSecret = corev1.SecretReference{Name: metallb.GetBGPPeerSecretName(p.Spec.ASN, p.Spec.Port, p.Spec.VRFName)}
 				p.Spec.Password = ""
+				p.Spec.ConnectTime = metav1.Duration{Duration: time.Second * 5}
```

Comment:
this change isn't necessary so the test will pass, but i found that is shortens the test in almost 2 minutes, and i think its nice so thought its worth mentioning it. I didn't give any thought if this can shorten more tests